chain/x/zoracle/internal/keeper/test_common.go

```diff
@@ -179,6 +179,14 @@ func newPubKey(pk string) (res crypto.PubKey) {
 	return pkEd
 }
 
+func GetAddressFromPub(pub string) (addr sdk.AccAddress, err error) {
+	return sdk.AccAddressFromHex(NewPubKey(pub).Address().String())
```

Comment:
sdk.AccAddress(NewPubKey(pub).Address()) after change it will not return anymore.